pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
-name = "nsidc-metgen"
-version = "0.1.0"
-description = "The nsidc-metgen package enables data producers as well as Operations staff managing the data ingest workflow to create metadata files conforming to NASA's Common Metadata Repository UMM-G specification."
+name = "nsidc-metgenc"
+version = "0.4.0"
```

Comment:
Does `bump-my-version` automatically update this value? (Oh, I really really hope so!)